src/deploy/osp_deployer/deployer.py

```diff
@@ -118,6 +118,10 @@ def deploy():
 
         logger.debug("loading settings files " + args.settings)
         settings = Settings(args.settings)
+        validate_performance_optimization_params(
```

Comment:
can be removed from here and incorporated in verify_deployer_settings, see comment below

Reply:
Validations moved to profile.py
Fixed